src/components/Account/TrustlineList.tsx

```diff
@@ -68,7 +69,9 @@ function TrustedAsset(props: TrustedAssetProps) {
           <SingleBalance
             assetCode=""
             balance={balance.balance}
-            style={isSmallScreen ? { fontSize: "1.1rem" } : { fontSize: "1.6rem" }}
+            style={
+              isTinyScreen ? { fontSize: "0.9rem" } : isSmallScreen ? { fontSize: "1.1rem" } : { fontSize: "1.6rem" }
```

Comment:
Would make more sense to make it `{ fontSize }` and define the fontSize above: `const fontSize = isTinyScreen ? "0.9rem" : ...`